CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# 11/15/2017 - v5.0.0
```

Comment:
We maintain the change log for CASino using GitHub's [release page](https://github.com/rbCAS/CASino/releases).